rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/TaskanaEngineController.java

```diff
@@ -97,13 +101,18 @@ public ResponseEntity<Map<String, List<String>>> getClassificationCategoriesByTy
    * This endpoint computes all information of the current user.
    *
    * @return the information of the current user.
+   * @throws InvalidNameException if the dn is invalid.
    */
   @GetMapping(path = RestEndpoints.URL_CURRENT_USER)
   @Transactional(readOnly = true, rollbackFor = Exception.class)
-  public ResponseEntity<TaskanaUserInfoRepresentationModel> getCurrentUserInfo() {
+  public ResponseEntity<TaskanaUserInfoRepresentationModel> getCurrentUserInfo()
+      throws InvalidNameException {
     TaskanaUserInfoRepresentationModel resource = new TaskanaUserInfoRepresentationModel();
     resource.setUserId(currentUserContext.getUserid());
-    resource.setGroupIds(currentUserContext.getGroupIds());
+    Map<String, List<String>> groupsAndPermissions =
```

Comment:
Should we extend currentUserContext so that we can also get Permissions? Or is it enough to only return groups but not permissions for the currentUser entpoint?